lib/hammer_cli_foreman_docker/docker_manifest.rb

```diff
@@ -2,16 +2,16 @@
   require 'hammer_cli_katello'
 
   module HammerCLIForemanDocker
-    class DockerImageCommand < HammerCLIForeman::Command
-      resource :docker_images
-      command_name 'image'
-      desc _('Manage docker images')
+    class DockerManifestCommand < HammerCLIForeman::Command
+      resource :docker_manifests
+      command_name 'manifest'
+      desc _('Manage docker manifests')
 
       class ListCommand < HammerCLIKatello::ListCommand
         include ::HammerCLIKatello::LifecycleEnvironmentNameResolvable
         output do
           field :id, _("ID")
-          field :image_id, _("Image ID")
+          field :manifest_id, _("Manifest ID")
           field :size, _("Size")
```

Comment:
Similar to the comment below, the output for the list command needs to be adapted to the information returned for a manifest.

Eg. size is not available, a manifest can only have 1 tag, that tag has a name...etc.